pkg/injector/go/windows/native/local_ntqueueapcthreadex.go

```diff
@@ -32,10 +32,10 @@ func (i *NtQueueApcThreadExLocal) GetImports() []string {
 
 func (e *NtQueueApcThreadExLocal) RenderInstanciationCode(data embed.FS) (string, error) {
 
-	return common.CommonRendering(data, "templates/go/injector/windows/native/local/NtQueueApcThreadEx-Local/instanciation.go.tmpl", e)
+	return common.CommonRendering(data, "templates/go/injector/windows/native/local/ntqueueapcthreadex/instanciation.go.tmpl", e)
 }
 
 func (e *NtQueueApcThreadExLocal) RenderFunctionCode(data embed.FS) (string, error) {
 
-	return common.CommonRendering(data, "templates/go/injector/windows/native/local/NtQueueApcThreadEx-Local/functions.go.tmpl", e)
+	return common.CommonRendering(data, "templates/go/injector/windows/native/local/createThread/functions.go.tmpl", e)
```

Comment:
Should be "templates/go/injector/windows/native/local/ntqueueapcthreadex/functions.go.tmpl"

Reply:
yup🤦